docs/cypress.config.js

```diff
@@ -5,5 +5,6 @@ module.exports = defineConfig({
     setupNodeEvents(on, config) {
       // implement node event listeners here
     },
+    chromeWebSecurity: false, //Skip Same Origin Policy to support tests that span multiple domains
```

Comment:
Added `chromeWebSecurity: false` to the [cypress.config.js](https://github.com/airqo-platform/AirQo-frontend/blob/staging/docs/cypress.config.js) to skip Same Origin Policy to that blocks tests that span multiple domains .

 [References](https://reflect.run/articles/accessing-a-new-window-in-cypress-tests/)